conf/modules.config

```diff
@@ -481,6 +481,17 @@ process {
             mode: params.publish_dir_mode,
         ]
     }
+
+    withName: GATK4_COLLECTREADCOUNTS {
```

Comment:
Could you expand the name to include the name of the subworkflow as well? 

Should be something like, '.*GENS:GATK4_COLLECTREADCOUNTS'. This is just to be consitent with the modules in this config.